README.md

```diff
@@ -19,10 +19,14 @@ Add the plugin to your build with the following in project/plugins.sbt:
 addSbtPlugin("com.sksamuel.scapegoat" %% "sbt-scapegoat" % "1.1.1") // Verify latest in badge above
 ```
 
-Then in your `build.sbt`, set the version of scapegoat you wish to use eg:
+The plugin has a default Scapegoat version depending on the Scala version you are using.
+This version may not always be the latest version of Scapegoat, nor always pick a compatible version.
```

Comment:
There's some language glitch in this line. Not sure I can suggest the best fix.

Reply:
Ah nice catch. ChatGPT says it should be `, nor does it always pick` which flows better.